support/magicleap/Servo2D/code/src/main.cpp

```diff
@@ -5,6 +5,7 @@
 #include <Servo2D.h>
 #include <ml_lifecycle.h>
 #include <ml_logging.h>
+#include <gst/gst.h>
```

Comment:
I think this is useless since you're not using gstreamer in this file.

Reply:
Ah, that's a holdover from the previous code for initializing gstreamer.